data_types/const_number.py

```diff
@@ -3,11 +3,11 @@
 class const_number(object):
```

Comment:
If we're already dropping 2.X compatibility, we should also stop inheriting from `object` since that is a Python 2 thing that does nothing in modern versions of python.